code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1499,6 +1499,19 @@
 		var/t_loc = get_turf(O)
 		qdel(O)
 		new /obj/item/clothing/shoes/galoshes/dry(t_loc)
+		
+/datum/reagent/americajuice
+	name = "America Juice"
```

Comment:
```suggestion
	name = "Freedom Juice"
```
Here is a "suggestion" on the name.
I'm not a part of the lore team but I think America doesn't exist in ss13